test/schema.test.py

```diff
@@ -196,6 +196,25 @@ def test_generate_output_columns_choice(self):
             schema1.generate_output_columns(),
         )
 
+    def test_drop_special_char_columns(self):
+        schema1 = Schema()
+        schema1.read_object({"abc ": 1, "def@#": 1, "$$ghi": 1, "jkl": 1, "!@#mno": 1})
+        schema1.drop_special_char_columns()
+        print(schema1.schema)
+        ##self.assertEqual(3, )
+        self.assertEqual(schema1.schema, {"abc ": "int", "jkl": "int"})
```

Comment:
with unit testing usually you want to also test the null case (in this case this would be a schema with no columns that should be dropped)